MapboxCoreNavigation/EventsManager.swift

```diff
@@ -74,68 +74,92 @@ open class EventsManager: NSObject {
         manager.sendTurnstileEvent()
     }
     
-    func navigationCancelEvent(rating potentialRating: Int? = nil, comment: String? = nil) -> EventDetails {
+    func navigationCancelEvent(rating potentialRating: Int? = nil, comment: String? = nil) -> EventDetails? {
+        guard let routeController = routeController else {
+            return nil
+        }
+        
         let rating = potentialRating ?? MMEEventsManager.unrated
         var event = EventDetails.defaultEvents(router: routeController)
```

Comment:
Can this be moved up to the guard statement in order to avoid the `?`'s below?

Reply:
sure can.